app/src/main/java/com/github/se/orator/ui/friends/FriendsScreen.kt

```diff
@@ -147,12 +160,12 @@ fun ViewFriendsScreen(
                       Box(
                           modifier =
                               Modifier.padding(vertical = AppDimensions.paddingMediumSmall)) {
-                            OutlinedTextField(
+                            androidx.compose.material.OutlinedTextField(
                                 value = searchQuery,
                                 onValueChange = { searchQuery = it },
-                                label = { Text("Search for a friend.") },
+                                label = { androidx.compose.material.Text("Search for a friend.") },
```

Comment:
Also here, I think it would be better to fix the library import and directly use Icon(...)